tests/push/test_items.py

```diff
@@ -3,7 +3,7 @@
 from pathlib import Path
 
 from gftools.push.items import Family, FamilyMeta, Designer, Axis, AxisFallback
-import importlib.resources
+from importlib.resources import files, as_file
```

Comment:
you didn't use as_file? I think you should even if it seems it's doing nothing, it would matter when the data files is coming from a zip